spec/supabase_dart_v1.yml

```diff
@@ -1382,7 +1382,7 @@ functions:
           final data = await supabase
             .from('cities')
             .select('name, country_id')
-            .order('id',  { ascending: false });
+            .order('id', ascending: true);
```

Comment:
Changed it to `ascending: true` instead of false, because ascending false is the default value for Flutter SDK.